wherehows-web/app/components/dataset-aclaccess.js

```diff
@@ -0,0 +1,138 @@
+import Component from '@ember/component';
+import { computed, getProperties, setProperties, get, set } from '@ember/object';
+import { requestAclAccess } from 'wherehows-web/utils/api/datasets/acl-access';
+import { accessState, pickList } from 'wherehows-web/constants/dataset-aclaccess';
+
+export default Component.extend({
+  /**
+   * Define property to bind textarea
+   * @type {string}
+   */
+  requestReason: '',
+
+  /**
+   * Define the property for the initial page  
+   * @type {object} 
+   */
+  accessInfo: null,
+
+  /**
+   * Define the property for reloading the page when a user applies for permission.
+   * @type {object} 
+   */
+  accessResponse: null,
+
+  /**
+   * Define the property to identify user 
+   * @type {string} 
+   */
+  currentUser: null,
```

Comment:
You've documented this `currentUser` property as a `string`, but initialized it to `null`? Seems like a discrepancy